packages/client/src/utils/validate.ts

```diff
@@ -260,6 +260,17 @@ export const minAgeGapExist = (
   return diff >= minAgeGap
 }
 
+export const isValidDate: Validation = (value: IFormFieldValue, drafts?) => {
+  const cast = value as string
+  return !cast
+    ? { message: messages.required }
```

Comment:
@wilrona won't this make any field having this validation required?

Reply:
I did a test and I didn't see any message telling me that the field is mandatory.

I just copied the isValidBirthDate function which is right after it. I think that if I remove it too it won't have a problem when I look at it.